examples/selenoid-example/browsers.json

```diff
@@ -0,0 +1,47 @@
+{
```

Comment:
Just curious, what if we don't have this file. Will a plugin work?

Reply:
@DavertMik Without this selenoid container cannot be created

Reply:
We need to figure out how to get start with Selenoid without asking user to create this file manually.
It's hard to explain to users how and where they should take this file...

Reply:
We can create the file from plugin itself, if it's not present. Your views?

Reply:
@DavertMik  I also mentioned where we can take this json 

> * ### 1. Add browsers.json
 >* Add browsers.json parallel to codecept conf location. [Refer here](https://aerokube.com/selenoid/latest/#_prepare_configuration) to know more about browsers.json
 >* **You can download the sample json from example.**

Reply:
Ok, I contacted Selenoid maintainer, he says that we can actually use `latest` as tag for browsers.
At this moment we need a minimal but functional self-updating config. We can have it like this:

```json
{
    "chrome": {
        "default": "latest",
        "versions": {
            "latest": {
                "image": "selenoid/chrome:latest",
                "port": "4444",
                "path": "/"
            }
        }
    }
```

Please try and check if it works for you. If it works just create this file by a plugin on "autocreate" if no such file exist.

Reply:
Having `latest` may create issue as it may not be stable. [Refer this issue thread](https://github.com/aerokube/selenoid/issues/801). It is better to have a stable version.
Your views?

Reply:
totally agreed :)